src/components/auth/signup/signup-modal.tsx

```
@@ -0,0 +1,69 @@
+import { Button, Checkbox, Input, Spacing } from "@/components/ui";
+import s from "./styles.module.scss";
+import { useState } from "react";
+
+export default function SignupModal() {
+    const [isTermsAgreed, setIsTermsAgreed] = useState(false);
+    const [isNewsletterAgreed, setIsNewsletterAgreed] = useState(false);
+
+    const handleSignup = () => {
+        if (!isTermsAgreed) {
```

ESLint

[error] Delete `····`

(prettier/prettier)